templates/enterprise-license-job.yaml

```
@@ -91,8 +91,8 @@ spec:
                 EOF
                 chmod +x ./apply-license.sh
 
-                # Time out after 20 minutes.
-                timeout -t 1200 ./apply-license.sh
+                # Time out after 20 minutes. Use || to support new timeout versions that dont accept -t
+                timeout -t 1200 ./apply-license.sh || timeout 1200 ./apply-license.sh
```

I think this might cause users confusion if the license apply fails because when they look at the logs, they'll read:
```
timeout: unrecognized option: t
BusyBox v1.31.1 () multi-call binary.

Usage: timeout [-s SIG] SECS PROG ARGS

Runs PROG. Sends SIG to it if it is not gone in SECS seconds.
Default SIG: TERM.
Applying license...
some-error
Retrying in 2s...
some-error
Retrying in 2s...
```

Can we suppress the usage output?

---

How about this?
```suggestion
                timeout -t 1200 ./apply-license.sh &> /dev/null || timeout 1200 ./apply-license.sh
```
Example: 
```
ls -l /tmp/nonexistantfile &> /dev/null || ls -l /tmp/existantfile
-rw-r--r--  1 kyle  wheel  0 Oct 19 18:47 /tmp/existantfile
```